src/editor/Editor.css

```diff
@@ -1,94 +1,75 @@
 @import url('https://fonts.googleapis.com/css?family=Kulim+Park&display=swap');
 
-.editor-container{
-    height: 100%;
+.editor-container {
     display: flex;
     flex-direction: column;
-
+    height: 100%;
 }
 
-.splitted-editor{
-    height: 100%;
+.splitted-editor {
     display: flex;
     flex-direction: column;
-} 
+    height: 100%;
+}
 
-.editor{
+.editor {
     display: flex;
-    
     flex-direction: row;
     flex: 1;
 }
 
-.ace-editor{
-    width: 50%;      
-    flex: 1;
-
+.ace-editor, .preview {
     display: flex;
-    flex-direction: column;
-    
-    border-left: 0px;
-    border-bottom: 0px;
-    border: 1px solid rgba(186, 186, 186, 0.53);
-    padding-left:0!important;
-    padding-right:0!important;
-}
-
-.preview{
     flex: 1;
-
-    display: flex;
     flex-direction: column;
-
+    width: 50%;
     border: 1px solid rgba(186, 186, 186, 0.53);
-    border-bottom: 0px;
-    padding-left:0!important;
-    padding-right:0!important;    
+    padding-left: 0 !important;
+    padding-right: 0 !important;
 }
 
-
-.markdown-content{
+.markdown-content {
     display: flex;
-
 }
 
-.markdown-body{
+.markdown-body {
     position: absolute;
     padding-left: 15px;
-    padding-right: 15px;  
+    padding-right: 15px;
 }
 
 .button-style {
+    width: 122px;
     height: 31px;
     margin-left: 10px;
-    width: 122px;
+}
+
+.button-toolbar {
+    align-items: baseline;
 }
 
 .document-name {
     font-style: italic;
-    border-top:0!important;
-    border-left:0!important;;
-    border-right:0!important;;
-    border-radius:0!important;;
+    border: 0 !important;
```

Comment:
@leventdeniz ,
you accidentally removed the border-bottom of the input field. 
Please revert that.